CMSgov/onemac
# Oy2 22984 - update attachment text language

Story: https://qmacbis.atlassian.net/browse/OY2-22984
Endpoint: See github-actions bot comment

### Details

Setup common language in JSX to be used as default and as part of any overridden attachmentsIntroText JSX block

### Changes

- updated formlib to have common JSX
- updated usage of attachmentsIntroJSX to use common JSX block for this section of text

### Implementation Notes

- Im not sure how useful this way will be in the future if say they wanted the FAQ link to point to a specific section of the FAQ per form type (similar to how the ID format FAQ Link works). May have to just completely override each attachmentsIntroText definition

### Test Plan

1. Login as state submitter
2. Navigate to each submission form and verify text.
